index.js

```diff
@@ -152,7 +156,12 @@ exports.parseFullName = function parseFullName(
       'vice admiral','viscount','viscountess','wg cdr', 'ind', 'misc', 'mx'];
   } else {
     suffixList = ['esq','esquire','jr','jnr','sr','snr','2','ii','iii','iv',
-      'md','phd','j.d.','ll.m.','m.d.','d.o.','d.c.','p.c.','ph.d.'];
+      'v','clu','chfc','cfp','md','phd','j.d.','ll.m.','m.d.','d.o.','d.c.',
+      'p.c.','ph.d.','ba','b.a.','beng','b.eng.','bsc','b.sc.', 'di','dipl.-ing.',
+      'mag. med. vet.','dr. med. dent.','mag. rer. nat.','mag. phil.', 'mag. iur.',
+      'm.a.i.s.','llm','ll.m.','msc','m.sc.','phd','dr. nat. techn.','dr. scient. med.',
+      'dr. mont.','dr. rer. nat.','dr. phil.','dr. iur.','dr. rer. soc. oec.','dr. theol.',
+      'm.e.s.','mas','ma','mba','m.b.l.','mib','mp','msc'];
```

Comment:
Let's constantize these list subsets so (a) it's clear that it's repeated in both places; and (b) if we need to update them, we only need to update in one place.

nits:
- Include in the description of your PR where you're sourcing your academic grades so we can use it as a reference in the future.
- Comment the list of academic grades with what they stand for (e.g., `'b.a.' // Bachelor of Arts`)

Reply:
https://www.bmbwf.gv.at/en/Topics/Higher-education---universities/Studying/Recognition-of-qualifications/Academic-degrees.html

Reply:
i added  academic grades  comments